modules/tests/src/smithy4s/tests/PizzaSpec.scala

```diff
@@ -463,6 +463,106 @@ abstract class PizzaSpec
       }
   }
 
+  routerTest("Respects Static Query Parameters") { (client, uri, log) =>
```

Comment:
I've played with different variations of static query params, but it seems operations are shuffled during code generation, so it's impossible to guarantee any order., i.e., overlapping routes can't have a predictable outcome.

Reply:
Can you add them nevertheless, and mark them as flaky with a mention of https://github.com/disneystreaming/smithy4s/issues/1567 ? That way, we'll know to fix them once the other thing is resolved